tests/Integration/Behaviour/Features/Scenario/Search/add_alias.feature

```diff
@@ -0,0 +1,24 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s alias --tags add
+@restore-aliases-before-feature
+@clear-cache-before-feature
+@clear-cache-after-feature
+@add
+Feature: Add basic alias from Back Office (BO)
+  As a BO user
+  I need to be able to add new alias with basic information from the BO
+
+  Scenario: I add an alias with basic information
+    When I add alias "alias1" with following information:
+      | alias   | alias1  |
+      | search  | alias 1 |
+    Then alias "alias1" should have the following details:
+      | alias   | alias1  |
+      | search  | alias 1 |
+
+  Scenario: I add a multiple aliases with basic information
```

Comment:
I think you should also add another use case where you add new aliases for the same search value, and if I got the behavior right when you assert values after they should have been added to the previous one no?

Or should they have replaced the initial ones and deleted them? I'm not sure what the expected behaviour is right now

Reply:
After grooming with @MatShir and @zuk3975 , we decided that it would be a better option to append the new aliases for to the search term.